### PR TITLE
Prepare next dev version

### PR DESCRIPTION
**Pull request by adambkaplan**
# Changes

Update next development version to v0.8.1-snapshot.

/kind cleanup

# Submitter Checklist

- [ ] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [x] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)  
- [x] Release notes block has been filled in, or marked NONE

See [the contributor guide](https://github.com/shipwright-io/build/blob/master/CONTRIBUTING.md)
for details on coding conventions, github and prow interactions, and the code review process.

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.8.0
+VERSION ?= 0.8.1-snapshot
```
Comment (gabemontero) [inline Makefile:6]: jenkins-esque :-)

**Review by gabemontero — APPROVED** [review]
/lgtm

**Comment by adambkaplan** [comment]
/approve

Self-approving. The e2e tests are currently failing because we're using the outdated `opm index add` command. This is deprecated, and may no longer be functional.

**Comment by adambkaplan** [comment]
/close

Made obsolete by https://github.com/shipwright-io/operator/pull/67

**Pull request closed without merging**